skyportal/handlers/api/spectrum.py

```diff
@@ -36,6 +36,19 @@
 
 
 class SpectrumHandler(BaseHandler):
+    def get_groups(self, group_ids):
+        groups = None
+        if group_ids:
```

Comment:
Is there ever a case now when group_ids can be None? If not you can simplify

Reply:
Fixed